File: app/views/helpers/will_paginate.rb
module ViewHelpers
  def previous_page
    if (self["current_page"] != 1)
      previous_page = (self["current_page"] == 2) ? nil : self["current_page"] - 1
      '<p class="left"><a href="/blog/' + previous_page.to_s + '">&larr; Previous page</a></p>'
    end
  end

  def next_page
    if (self["current_page"] != self["total_pages"])
      '<p class="right"><a href="/blog/' + (self["current_page"] + 1).to_s + '">Next page &rarr;</a></p>'
    end
  end
end

File: public/js/helpers/will_paginate.js
/* Author: 
- Félix Bellanger <felix.bellanger@gmail.com>
*/

Handlebars.registerHelper('previous_page', function() {

  if (this.current_page != 1) {
    return '<p class="left"><a href="/blog/' + (this.current_page == 2) ? null : (this.current_page - 1) + '">&larr; Previous page</a></p>';
  }

});

Handlebars.registerHelper('next_page', function() {

  if (this.current_page != this.total_pages) {
    return '<p class="right"><a href="/blog/' + (this.current_page + 1) + '">Next page &rarr;</a></p>';
  }

});

